arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/AccountSnapshot.java

```diff
@@ -156,17 +156,10 @@ public AccountSnapshot deepCopy() {
         address, nonce, balance, isWarm, code, deploymentNumber, deploymentStatus);
   }
 
-  public AccountSnapshot wipe(DeploymentInfo deploymentInfo) {
+  public void wipe(DeploymentInfo deploymentInfo) {
     final boolean deploymentStatus = deploymentInfo.getDeploymentStatus(address);
     checkArgument(!deploymentStatus);
-    return new AccountSnapshot(
-        address,
-        0,
-        Wei.of(0),
-        isWarm,
-        Bytecode.EMPTY,
-        deploymentInfo.deploymentNumber(address),
-        deploymentStatus);
+    this.nonce(0).balance(Wei.ZERO).code(Bytecode.EMPTY).setDeploymentInfo(deploymentInfo);
```

Comment:
The `wipe` method must be invoked after the `SELFDESTRUCT`'ing account's deployment information underwent a `freshDeployment`.